hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -992,4 +1005,72 @@ objects:
                 lastSyncTimestamp: remove
                 storageBucket:
                   lastSyncTimestamp: remove
-
+          exposeAsMetric:
+            - eventselector:
+                users:
+                  usernames: ["!system:*"]
+                verbs: ["delete", "update", "patch"]
+                objectReferences:
+                  - names: ["sre-*", "*openshift.io"]
+                    resources: ["validatingwebhookconfigurations"]
+                responseStatus:
+                  codes: ["!403"]
+              alert: NonSystemChangeValidationWebhookConfiguration  
+    - apiVersion: v1
+      kind: Service
+      metadata:
+        labels:
+          app: audit-exporter
+        name: audit-exporter
+        namespace: openshift-security
+        annotations:
+          service.beta.openshift.io/serving-cert-secret-name: audit-exporter-certs
+      spec:
+        ports:
+          - name: metrics
+            port: 9090
+            protocol: TCP
+            targetPort: 9090
+        selector:
+          app: audit-exporter
+        sessionAffinity: None
+        type: ClusterIP
+    - apiVersion: monitoring.coreos.com/v1
+      kind: ServiceMonitor
+      metadata:
+        name: audit-exporter
+        namespace: openshift-security
+      spec:
+        endpoints:
+          - path: /metrics
+            port: metrics
+            scheme: https
+            tlsConfig:
+              caFile: /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
```

Comment:
TIL - Thanks for this, it is a really great idea, ref: https://docs.openshift.com/container-platform/4.11/security/certificate_types_descriptions/service-ca-certificates.html